backend/infrahub/services/adapters/message_bus/__init__.py

```diff
@@ -26,20 +29,35 @@ class InfrahubMessageBus:
     ]
     event_bindings: list[str] = ["refresh.registry.*"]
     broadcasted_event_bindings: list[str] = ["refresh.git.*"]
+    service: InfrahubServices
 
-    async def initialize(self, service: InfrahubServices) -> None:
-        """Initialize the Message bus"""
-
+    @abstractmethod
     async def shutdown(self) -> None:
-        """Shutdown the Message bus"""
+        pass
```

Comment:
For situations like this I think it's typically better to keep the doc string as it would still be a valid function. The main reason is that a `pass` can show up as a line that isn't covered by tests where as a doc string doesn't.